apisix/discovery/nacos.lua

```diff
@@ -275,41 +273,33 @@ end
 
 
 function _M.nodes(service_name)
-    local logged = false
```

Comment:
The original code is good. We just need to set a 5s limitation.

Reply:
Just keep simple.

Reply:
The lazy init is important as the sync is slow.

Reply:
OK,i rollback the code.